app/src/main/res/layout-sw600dp-port/onboarding_slide_final.xml

```diff
@@ -44,8 +44,7 @@
         android:gravity="center_horizontal"
```

Comment:
Apply all the changes to this file which I have mentioned previously.

Reply:
Made the change!